app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -257,6 +270,41 @@ class Core(private val context: Context) {
         }
     }
 
+    private fun getCookiePolicy(): CookiePolicy {
+        return if (context.settings().blockCookiesInCustomTrackingProtection) {
+            when (context.settings().blockCookiesSelectionInCustomTrackingProtection) {
+                "all" -> CookiePolicy.ACCEPT_NONE
+                "social" -> CookiePolicy.ACCEPT_NON_TRACKERS
+                "unvisited" -> CookiePolicy.ACCEPT_VISITED
+                "third-party" -> CookiePolicy.ACCEPT_ONLY_FIRST_PARTY
+                else -> CookiePolicy.ACCEPT_NONE
+            }
+        } else {
+            CookiePolicy.ACCEPT_ALL
+        }
+    }
+
+    private fun getTrackingCategories(): Array<TrackingCategory> {
```

Comment:
We could also add custom to this fun `getCustomTrackingCategories` to avoid misuses